server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyRequestBody.scala

```diff
@@ -0,0 +1,32 @@
+package sttp.tapir.server.netty
+
+import scala.concurrent.Future
+
+import io.netty.handler.codec.http.{FullHttpMessage, FullHttpRequest, HttpContent}
+import sttp.capabilities
+import sttp.tapir.RawBodyType
+import sttp.tapir.server.interpreter.{RawValue, RequestBody}
+import scala.concurrent.ExecutionContext.Implicits.global
+
+import io.netty.buffer.ByteBufInputStream
+import sttp.tapir.internal.NoStreams
+
+//todo: ChannelFuture or something with cats-effect?
+//todo: FullHttpMessage OK?
+class NettyRequestBody(req: FullHttpMessage) extends RequestBody[Future, NoStreams] {
```

Comment:
going forward we probably won't want to use `FullHttpMessage`, as if I understand correctly this reads the whole request content into memory. For files & streams, we don't want this to happen - so probably another implementation will have to be used.